src/main/features/ccj/index.ts

```diff
@@ -27,11 +27,16 @@ export class CCJFeature {
     app.all(allCCJ, requestHandler())
     app.all(allCCJ, ClaimMiddleware.retrieveByExternalId)
     app.all(allCCJ, OnlyClaimantLinkedToClaimCanDoIt.check())
-    app.all(/^\/case\/.+\/ccj\/(?!confirmation).*$/, CCJGuard.requestHandler)
-    app.all(/^\/case\/.+\/ccj\/(?!confirmation).*$/,
+    app.all(/^\/case\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\/ccj\/(?!confirmation).*$/, CCJGuard.requestHandler)
```

Comment:
Since regex is the same we could remove that line and add `CCJGuard.requestHandler` to next one.

Reply:
Done